manifests/jenkins-operator.0.0.0.clusterserviceversion.yaml

```diff
@@ -103,7 +103,7 @@ metadata:
     capabilities: Basic Install
     categories: Developer Tools
     certified: "true"
-    containerImage: quay.io/redhat-developer/openshift-jenkins-operator:0.0.0
+    containerImage: quay.io/redhat-developer/openshift-jenkins-operator:latest
```

Comment:
can you use 0.0.0 per convention, so we are sure that we have versions alignment between, index, bundle and image.